Formula/balena-cli.rb

```diff
@@ -22,8 +22,6 @@ class BalenaCli < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "c2e6b54f260a8a8051ac73478f8f2d149552ddb41a0987978b0ac27e2038574e"
   end
 
-  # Node looks for an unversioned `python` at build-time.
-  depends_on "python@3.10" => :build
```

Comment:
I'm updating `balena-cli` here as it helps test out that PATH works and allows us to drop this indirect dependency.